src/main/java/net/sf/rails/game/financial/ShareSellingRound.java

```diff
@@ -54,7 +58,7 @@ public void start(RoundFacade parentRound, Player sellingPlayer, int cashToRaise
         this.cashToRaise = IntegerState.create(this, "CashToRaise", cashToRaise);
 
         this.dumpOtherCompaniesAllowed = dumpOtherCompaniesAllowed;
-        log.debug("Forced selling, dumpOtherCompaniesAllowed = {}", dumpOtherCompaniesAllowed);
+        log.debug("Forced selling, dumpOtherCompaniesAllowed = " + dumpOtherCompaniesAllowed);
```

Comment:
Looks like some of the logging variable changes were reverted?

Reply:
Sorry about that overlook